master/internal/experiment.go

```diff
@@ -481,7 +503,10 @@ func (e *experiment) processOperations(
 		case searcher.Create:
 			checkpoint, err := e.checkpointForCreate(op)
 			if err != nil {
-				e.updateState(ctx, model.StoppingErrorState)
+				e.updateState(ctx, model.StateWithReason{
+					State:               model.StoppingErrorState,
+					InformationalReason: "hp searcher unable to get checkpoint",
```

Comment:
include the trial id and error too? (since one trial failing here can crash the others)
```suggestion
					InformationalReason: "hp searcher unable to get checkpoint for new trial",
```